docs/content/users/install/ddev-installation.md

```diff
@@ -132,6 +138,9 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
         curl -fsSL https://ddev.com/install.sh | bash -s v1.21.4
         ```
 
+    ??? "Do you still have an old version after installing or upgrading?"
```

Comment:
I think it can also be copied after Debian/Ubuntu: because it's the most used Linux and people are unlikely to scroll down here.